recipes/coin/meta.yaml

```diff
@@ -0,0 +1,55 @@
+package:
+  name: coin
+  version: 4.0.0
+
+source:
+  fn: coin.tar.gz
+  url: https://bitbucket.org/Coin3D/coin/get/CMake.tar.gz
+  sha256: e93d77e6ac61f166d93b66e60c644928935263ded1c27205f2e4352adaafdf97
+
+  patches:
+    - cmake.patch
+    - windows.patch  # [win]
+    - windows1.patch  # [win]
+
+build:
+  number: 0
+  skip: true  # [win and not py35]
+
+  track_features:
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - cmake  # [unix]
+    - ninja  # [win]
+    - python  # [win]
+    - boost 1.61.*
+    - toolchain  # [osx]
+  run:
+    - boost 1.61.*
+
+
+test:
+  files:
+    - test.cpp  # [linux]
+    - test.sh  # [osx]
+  commands:
+    - g++ -o test test.cpp -lCoin -L${PREFIX}/lib/ -I${PREFIX}/include/  # [linux]
+    - ./test  # [linux]
+    - bash test.sh  # [osx]
+    - where /r "%PREFIX%"  Coin4.dll  # [win]
+
+about:
+  home: https://bitbucket.org/Coin3D/coin/wiki/Home
+  license: BSD 3-Clause License
```

Comment:
I haven't found one. Should I include a copy of BSD 3 license in the recipe-dir?